app/forms/claim/mentors_form.rb

```diff
@@ -0,0 +1,41 @@
+class Claim::MentorsForm < ApplicationForm
+  attr_accessor :claim, :mentor_ids
+
```

Comment:
I don't think this works as you expect, I had the same thought but the error does not come on the form. This is because I have a `to_model` method in here. This is for the edit action, so the form remembers the checkbox selections.


https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/5a522321-3e97-439f-a2ed-898fb0755464

Reply:
I don't think this is a good place to put this validation. I think validations like these are meant to guide the user to fill a form or do something correctly.

Checking the presence of a claim here seems like a validation for developers to make sure they use this class corectly. And if we do want something like this, we should probably do it earlier, in the initialization of the class rather than when we call the `save` method